.github/workflows/continuous-integration.yml

```diff
@@ -25,7 +25,7 @@ jobs:
                   node-version: 18.x
                   registry-url: https://registry.npmjs.org
             - name: Install packages
-              run: npm ci
+              run: npm ci --legacy-peer-deps
```

Comment:
**issue:** This is adding an exception in the install flow. Setting install options in the CI won't remove issues when installing locally. A better approach is to rely on a local `.npmrc` so that the experience is seamless for everyone.